spec/lib/convertkit/resources/subscribers_response_spec.rb

```diff
@@ -0,0 +1,39 @@
+describe ConvertKit::Resources::SubscribersResponse do
+  include Validators::SubscriberValidators
+
+  describe '#initialize' do
+    it 'set with an empty list of subscribers' do
+      response = { 'total_subscribers' => 0, 'page' => 1, 'total_pages' => 1, 'subscribers' => [] }
+      tag_response = ConvertKit::Resources::SubscribersResponse.new(response)
+      validate_subscribers(tag_response, response)
+    end
+
+    it 'set with a list of subscribers' do
+      response = {
```

Comment:
Curiosity question:
Is this response structure something agreed upon with them or is this something we have full control on and have decided internally?

Reply:
The responses are all based on the structure provided as example responses in the API documentation and so this is all set by ConvertKit.  

Here's an example from https://developers.convertkit.com/#list-subscribers.
<img width="533" alt="CleanShot 2023-09-05 at 11 36 07@2x" src="https://github.com/untitledstartup/convertkit-api-ruby/assets/50114720/73f111d8-61a6-44ee-8b71-636553f6ffe1">